pricegraph/src/orderbook.rs

```diff
@@ -578,11 +586,9 @@ impl Orderbook {
         Some((order, user))
     }
 
-    /// Returns `true` if the specified token pair is valid, and `false`
-    /// otherwise.
```

Comment:
nit: I would prefer to leave the doc comment in, maybe we can modify it to be more informative:
```rust
    /// Returns `true` if the specified token pair is valid, and `false`
    /// otherwise.
    ///
    /// A token pair is considered valid if both the buy and sell token
    /// exist in the current orderbook.
```